docs/docs/how-to/previews-deploys-hosting/deploying-to-vercel.md

```diff
@@ -2,7 +2,9 @@
 title: Deploying to Vercel
 ---
 
-[Vercel](https://vercel.com/home) is a cloud platform that enables developers to host Jamstack websites and web services that deploy instantly, scale automatically, and requires no supervision, all with zero configuration. They provide a global edge network, SSL encryption, asset compression, cache invalidation, and more.
+[Vercel](https://vercel.com/home) is the platform for frontend developers, providing the speed and reliability innovators need to create at the moment of inspiration. They support over 35 frontend frameworks, including Gatsby.
+
+Vercel supports Gatsby features like SSG, SSR, DSG, API Routes, and more.
```

Comment:
Does SSR and DSG support local images/files? (`File` nodes?)

Netlify currently has this limitation (https://github.com/netlify/netlify-plugin-gatsby#caveats) and I checked your source code and I'd assume it's currently not supported. Is that right?

Reply:
We mentioned Image CDN below, does Vercel support that? https://www.gatsbyjs.com/blog/image-cdn-lightning-fast-image-processing-for-gatsby-cloud/

Netlify supports it through this: https://github.com/netlify/netlify-plugin-gatsby/blob/main/docs/image-cdn.md

If it doesn't currently it should be mentioned :)

Reply:
Fixed!